elements/noflo-ui.html

```diff
@@ -506,6 +526,29 @@
           component: this.ctx.component,
           runtime: this.ctx.runtime
         });
+      },
+      addToList: function (list, entity) {
```

Comment:
This should not be a method in a Polymer element, but instead a free-standing function in some JS/CS file.